warehouse/views.py

```diff
@@ -101,6 +101,23 @@ def robotstxt(request):
 
 
 @view_config(
+    route_name="opensearch.xml",
+    renderer="opensearch.xml",
+    decorator=[
+        cache_control(1 * 24 * 60 * 60),         # 1 day
```

Comment:
This is pretty standard for how the rest of Warehouse does it. I don't have a strong opinion one way or another in general, but for this specific PR I think that consistency is better.